src/sbvr-api/permissions.ts

```diff
@@ -1273,6 +1273,102 @@ export const getUserPermissions = async (
 	}
 };
 
+export const getRolePermissions = async (
+	roleName: string,
+	userId: number,
+	tx?: Tx,
+): Promise<string[]> => {
+	try {
+		if (typeof userId === 'string') {
+			userId = parseInt(userId, 10);
+		}
```

Comment:
`userId` is typed as a number but you're also expecting a string? If so, I'd update the parameter type to reflect that or remove the string check + parseInt if you know you're getting a number.

Reply:
This was mostly just mimic from https://github.com/balena-io/pinejs/blob/master/src/sbvr-api/permissions.ts#L1134-L1148 but I agree, we should have the correct typing. My understanding is that this will always be numeric so I will remove the check actually